newIDE/app/src/PropertiesEditor/index.js

```diff
@@ -84,7 +84,7 @@ export default class PropertiesEditor extends React.Component<Props, {||}> {
     if (field.valueType === 'boolean') {
       return (
         <InlineCheckbox
-          label={field.name}
+          label={field.getName(this.props.instances[0])}
```

Comment:
So Flow typing of Field is very weak (it's just an Object) so Flow is not warning you but `getName` is not mandatory. For example, in InstancePropertiesEditor, it's not defined.

I suggest that we create a method, like _getFieldValue, called _getFieldLabel:

```
_getFieldValue(instances: Instances, field: Field): string {
  if (field.getLabel) return field.getLabel(instances[0]); //Maybe add a console.error if !instances[0], like in _getFieldValue. For some reason it happened according to some bug reports.

  return field.name;
}
```